components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -1205,32 +1132,21 @@ public DataSection disordersBody(Patient patient)
         Integer y = 0;
         for (Disorder disorder : disorders) {
             Integer x = 0;
-            if (present.contains("name")) {
-                DataCell cell = new DataCell(disorder.getName(), x, y);
-                cell.setMultiline();
-                bodySection.addCell(cell);
-                x++;
-            }
-            if (present.contains("id")) {
-                DataCell cell = new DataCell(disorder.getId(), x, y);
-                bodySection.addCell(cell);
-                x++;
-            }
+            DataCell cell = new DataCell(disorder.getName(), x++, y);
+            cell.setMultiline();
+            bodySection.addCell(cell);
+            bodySection.addCell(new DataCell(disorder.getId(), x, y));
             y++;
+            x++;
         }
         /* If there is no data, but there are headers present, create empty cells */
         if (disorders.isEmpty()) {
             Integer x = 0;
-            if (present.contains("name")) {
-                DataCell cell = new DataCell("", x, y);
-                bodySection.addCell(cell);
-                x++;
-            }
-            if (present.contains("id")) {
-                DataCell cell = new DataCell("", x, y);
-                bodySection.addCell(cell);
-                x++;
-            }
+            DataCell cell = new DataCell("", x, y);
+            bodySection.addCell(cell);
+            x++;
+            bodySection.addCell(new DataCell("", x, y));
+            x++;
         }
         /* Notes export */
         if (present.contains("notes")) {
```

Comment:
This `if` is now redundant, and actually leaving it in means that notes will not be displayed anymore.